[Bug]: The test is failed

### Brief problem description

we have 404 op code instead of 200 in the response

### Steps to Reproduce

uv run poe rest

### Expected Result

4 passed

### Actual Result

3 passed 1 failed
FAILED tests/test_items.py::test_get_item_1 - assert 404 == 200

--- a/.vscode/settings.json
+++ b/.vscode/settings.json
@@ -19,7 +19,7 @@
     "editor.defaultFormatter": "charliermarsh.ruff"
   },
 
-  "workbench.sideBar.location": "right",
+  "workbench.sideBar.location": "left",
 
   // Markdown settings
 

--- a/src/app/services/item_service.py
+++ b/src/app/services/item_service.py
@@ -228,7 +228,7 @@ def get_item_by_id_dfs_iterative(
 
                     for task in lab.tasks:
                         counter += 1
-                        if lab.id == item_id:
+                        if task.id == item_id:
                             return FoundItem(task, counter)
 
                         for step in task.steps:

